Client/Frontend/Fakespot/Views/FakespotAdView.swift

```diff
@@ -0,0 +1,315 @@
+            cardContainer.bottomAnchor.constraint(equalTo: footerLabel.topAnchor, constant: -UX.vStackSpacing),
+
+            footerLabel.leadingAnchor.constraint(equalTo: leadingAnchor),
+            footerLabel.trailingAnchor.constraint(equalTo: trailingAnchor),
+            footerLabel.bottomAnchor.constraint(equalTo: bottomAnchor),
+        ])
+
+        setupLayout()
+    }
+
+    private func adjustLayout() {
+        let contentSizeCategory = UIApplication.shared.preferredContentSizeCategory
+        starRatingHeightConstraint?.constant = min(UIFontMetrics.default.scaledValue(for: UX.starSize), UX.starMaxSize)
+        productImageHeightConstraint?.constant = min(UIFontMetrics.default.scaledValue(for: UX.productImageMinHeight), UX.productImageMaxHeight)
+        productImageWidthConstraint?.constant = min(UIFontMetrics.default.scaledValue(for: UX.productImageMinWidth), UX.productImageMaxWidth)
+
+        if contentSizeCategory.isAccessibilityCategory {
+            secondRowStackView.axis = .vertical
+            spacer.isHidden = false
+        } else {
+            secondRowStackView.axis = .horizontal
+            spacer.isHidden = true
+        }
+        setupLayout()
+        setNeedsLayout()
+        layoutIfNeeded()
+    }
+
+    // MARK: Subviews Setup
+    fileprivate func setupLayout() {
+        secondRowStackView.removeAllArrangedViews()
+        thirdRowStackView.removeAllArrangedViews()
+        contentStackView.removeAllArrangedViews()
+
+        let contentSizeCategory = UIApplication.shared.preferredContentSizeCategory
+        if contentSizeCategory.isAccessibilityCategory {
+            setupLayoutForAccessibilityView()
+        } else {
+            setupLayoutForDefaultView()
+        }
+    }
+
+    private func setupLayoutForDefaultView() {
+        // first line
+        contentStackView.addArrangedSubview(titleLabel)
+
+        // second line
+        productLinkButton.setContentHuggingPriority(.required, for: .vertical)
+        productLinkButton.setContentCompressionResistancePriority(.required, for: .vertical)
+        productLinkButton.setContentHuggingPriority(.required, for: .horizontal)
+        productLinkButton.setContentCompressionResistancePriority(.required, for: .horizontal)
+
+        secondRowStackView.addArrangedSubview(productImageView)
+        secondRowStackView.addArrangedSubview(productLinkButton)
+
+        secondRowStackView.addArrangedSubview(gradeReliabilityScoreView)
+        secondRowStackView.distribution = .fillProportionally
+        secondRowStackView.alignment = .top
+
+        // third line
+        thirdRowStackView.addArrangedSubview(priceLabel)
+        thirdRowStackView.addArrangedSubview(spacer)
+        thirdRowStackView.addArrangedSubview(starRatingView)
+        thirdRowStackView.distribution = .fill
+        thirdRowStackView.alignment = .leading
+
+        priceLabel.heightAnchor.constraint(equalTo: starRatingView.heightAnchor, multiplier: 1.0).isActive = true
```

Comment:
Labels should not have height constraints as this could lead to text being cut off when using large dynamic type.